src/i18n/locales/en-US.json

```diff
@@ -147,6 +147,14 @@
   "cmps.designer.link.Peer.implementation": "Implementation",
   "cmps.designer.link.Peer.version": "Version",
   "cmps.designer.link.Peer.status": "Status",
+  "cmps.designer.link.lndbackend.title": "Taro Lnd Connection",
+  "cmps.designer.link.lndbackend.desc": "Taro nodes communicate over lnd to mint and transfer assets.",
+  "cmps.designer.link.lndbackend.taroTitle": "Taro Node",
+  "cmps.designer.link.lndbackend.lndTitle": "Lnd Node",
```

Comment:
Rename "Lnd" to "LND". It should either be in all caps when referring to the name or all lower when referring to the CLI command.

Reply:
Done